docs/reference/config-params.md

```diff
@@ -380,26 +380,26 @@ services:
 ...
 ```
 
-## kev.workload.liveness-probe-disabled
+## kev.workload.liveness-probe-type
 
-Defines whether workload should have a liveness probe enabled. See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command). Kev will attempt to infer from the information specified in the compose file.
+Defines the workload's liveness probe type. See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command). Kev will attempt to infer from the information specified in the compose file.
 
 Kev uses the following heuristics to derieve that information for each service:
 
-If compose file(s) specifies the `healthcheck.disable` attribute key in a service config it will use its value.
-Otherwise it'll default to `false` (liveness probe active!)
+If compose file(s) specifies the `healthcheck.disable` attribute key in a service config it set the probe type to `none`.
+Otherwise it'll default to `command` (liveness probe active!)
 
-### Default: `false`
+### Default: `command`
 
-### Possible options: Bool
+### Possible options: none, command, http, tcp.
 
 > kev.workload.liveness-probe-disabled:
 ```yaml
 version: 3.7
 services:
   my-service:
     labels:
-      kev.workload.liveness-probe-disabled: true
+      kev.workload.liveness-probe-type: http
```

Comment:
For simplicity, should this be `....-type: none`? I ask, because we're not showing the expected related `http` probe labels.

Reply:
sure, I just thought having a "read example" could be good instead of copying and then changing it since realistically no one is going to add none 😛  but yes I get the point